core/admin.py

```diff
@@ -199,6 +200,12 @@ def view_add_organizers(self, request):
             'form': form,
             'title': 'Add organizers',
         })
+    
+    def save_model(self, request, obj, form, change):
+        created = not obj.pk
+        super(EventAdmin, self).save_model(request, obj, form, change)
+        if created:
+            form.add_default_data(obj)
```

Comment:
A little bit confused why we need this on both EventForm and EventAdmin, as EventAdmin is using EventForm anyway. There is probably a reason though, right?